package.json

```diff
@@ -36,52 +37,52 @@
     "main": "./main",
     "activationEvents": [
         "onLanguage:mongo",
-        "onView:cosmosDBExplorer",
-        "onCommand:cosmosDB.attachDatabaseAccount",
-        "onCommand:cosmosDB.attachEmulator",
-        "onCommand:cosmosDB.createAccount",
-        "onCommand:cosmosDB.deleteAccount",
-        "onCommand:cosmosDB.deletePostgresServer",
-        "onCommand:cosmosDB.detachDatabaseAccount",
-        "onCommand:cosmosDB.createMongoDatabase",
-        "onCommand:cosmosDB.createMongoCollection",
-        "onCommand:cosmosDB.createMongoDocument",
-        "onCommand:cosmosDB.createDocDBDatabase",
-        "onCommand:cosmosDB.createDocDBCollection",
-        "onCommand:cosmosDB.createDocDBDocument",
-        "onCommand:cosmosDB.createDocDBStoredProcedure",
-        "onCommand:cosmosDB.createGraphDatabase",
-        "onCommand:cosmosDB.createGraph",
-        "onCommand:cosmosDB.createPostgresDatabase",
-        "onCommand:cosmosDB.connectMongoDB",
-        "onCommand:cosmosDB.deleteMongoDB",
-        "onCommand:cosmosDB.deleteMongoCollection",
-        "onCommand:cosmosDB.deleteMongoDocument",
-        "onCommand:cosmosDB.deleteDocDBDatabase",
-        "onCommand:cosmosDB.deleteDocDBCollection",
-        "onCommand:cosmosDB.deleteDocDBDocument",
-        "onCommand:cosmosDB.deleteGraphDatabase",
-        "onCommand:cosmosDB.deletePostgresDatabase",
-        "onCommand:cosmosDB.deletePostgresTable",
-        "onCommand:cosmosDB.deleteGraph",
-        "onCommand:cosmosDB.executeAllMongoCommands",
-        "onCommand:cosmosDB.executeMongoCommand",
-        "onCommand:cosmosDB.importDocument",
-        "onCommand:cosmosDB.newMongoScrapbook",
-        "onCommand:cosmosDB.update",
-        "onCommand:cosmosDB.openDocument",
-        "onCommand:cosmosDB.openStoredProcedure",
-        "onCommand:cosmosDB.openPostgresFunction",
-        "onCommand:cosmosDB.deletePostgresFunction",
-        "onCommand:cosmosDB.openCollection",
-        "onCommand:cosmosDB.loadMore",
-        "onCommand:cosmosDB.refresh",
-        "onCommand:cosmosDB.openInPortal",
-        "onCommand:cosmosDB.copyConnectionString",
-        "onCommand:cosmosDB.launchMongoShell",
-        "onCommand:cosmosDB.openGraphExplorer",
-        "onCommand:cosmosDB.enterPostgresCredentials",
-        "onCommand:cosmosDB.configurePostgresFirewall"
+        "onView:azureDatabasesExplorer",
+        "onCommand:azureDatabases.attachDatabaseAccount",
```

Comment:
I'm nervous about changing all these command ids. It breaks people's custom keybindings since the command id gets hard-coded in their keybindings.json file (See "Open Keyboard Shortcuts (JSON)" in the command palette). For example, we default `cosmosDB.executeMongoCommand` to `cmd+shift+'` for Mac, but maybe they changed it to something else.

How about we keep `cosmosDB.` as the prefix for cosmos-specific commands, use `postgreSQL.` for the prefix for any postgres-specific commands, and use `azureDatabases.` for any commands that apply to both. This _kind of_ follows the same pattern we use for the command category. One note assuming we do that: "postgreSQL.deletePostgresFunction" feels redundant, so I think I'd prefer the pattern "postgreSQL.deleteFunction"